docs/GettingStartedOnLinux.md

```diff
@@ -0,0 +1,217 @@
+---
+id: getting-started
+title: Getting Started on Linux
+layout: docs
+category: Quick Start
+permalink: docs/getting-started-on-linux.html
+next: android-setup
+---
+
+## Intro
+
+This guide is essentially a beginner-friendly version of the [Getting Started](/react-native/docs/getting-started.html) page for React Native on Linux.
+
+### Platform
+
+For the purposes of this guide, we assume that you're working on Ubuntu Linux 14.04 LTS.
+
+### Prerequisites
+
+Before following this guide, you should have installed the Android SDK and run a successful Java-based "Hello World" app for Android.
+
+Other than that, we should have you covered!
+
+Let's dig in.
+
+### Installing everything
+
+#### Install NodeJS
+
+The first thing you need to do is to install NodeJS, a popular Javascript implementation.
+
+Fire up a terminal and paste these commands in to install NodeJS from APT:
+
+* `sudo apt-get install -y build-essential`
+* `curl -sL https://deb.nodesource.com/setup_4.x | sudo -E bash -`
+* `sudo apt-get install -y nodejs`
+
+#### Install NPM
+
+After installing NodeJS, you should now have NPM, too, the Node Package Manager.
+
+You can check by typing `which npm` into the terminal.
+
+If have NPM, you'll see something like:
+
+```bash
+$ /usr/local/bin/npm
+```
+
+If you don't have NPM, you'll see:
+
+```bash
+$ which npm
+npm not found
+$
+```
+
+If that's the case, run the following command to install NPM:
+
+`curl http://npmjs.org/install.sh | sh`
+
+#### Install watchman
+
+[watchman](https://facebook.github.io/watchman/docs/install.html) is a tool developed by Facebook for watching the filesystem. For best results, you'll want to install it.
+
+Once again, paste these into your terminal to get up and running:
```

Comment:
`Paste the following into your Terminal to compile watchman from source and install.`